avro2tf/src/main/scala/com/linkedin/avro2tf/jobs/Avro2TF.scala

```diff
@@ -88,6 +88,14 @@ object Avro2TF {
    */
   case class SparseVector(indices: Seq[Long], values: Seq[Float])
 
+  /**
+    * Case class for ranked [[SparseVector]] type. Ranked SparseVector is the format of a list of SparseVector,
+    * to represent it, we will have a 2D indices array and a 1D values array
+    * @param indices 2D indices array, which has indices(0) as first dimension of value and indices(1) as second dimension of value
+    * @param values 1D values array, which has concatenation of all values from original SparseVector values
+    */
+  case class RankedSparseVector(indices: Seq[Seq[Long]], values: Seq[Float])
```

Comment:
I also suggest give it a different name, e.g. `MultiDimSparseVector`.

Reply:
Yes, renamed as you suggested in another pull request (https://github.com/linkedin/Avro2TF/pull/62) I will close this pull request